supla-common/proto.h

```diff
@@ -1320,12 +1322,64 @@ typedef struct {
 #define RS_VALUE_FLAG_MOTOR_PROBLEM 0x8
 #define RS_VALUE_FLAG_CALIBRATION_IN_PROGRESS 0x10
 
-typedef struct {
-  char position;  // -1 == calibration. -1 - 100%
-  char tilt;
-  char bottom_position;  // Percentage points to the windowsill
-  _supla_int16_t flags;
-} TRollerShutterValue;
+#define SUPLA_FACADEBLIND_TYPE_KEEP_POSITION_DURING_TILTING 1
+#define SUPLA_FACADEBLIND_TYPE_CHANGE_POSITION_DURING_TILTING 2
+#define SUPLA_FACADEBLINE_TYPE_TILTING_ONLY_AT_FULLY_CLOSED 3
+
+// Roller shutter channel value payload
+// Device -> Server -> Client
+typedef struct {
+  signed char position;  // -1 == calibration. -1 - 100%, DSC
+  char reserved1;
+  signed char bottom_position;  // Percentage points to the windowsill, SC
+  _supla_int16_t flags;  // DSC
+  char reserved2;
+  char reserved3;
+  char reserved4;
+} TDSC_RollerShutterValue;
```

Comment:
All new structures must be concreted here:
https://github.com/SUPLA/supla-core/blob/develop/supla-common/test/ProtoTest.cpp#L211
ASSERT_EX(sizeof(TDSC_RollerShutterValue), (unsigned int)X);

Structures which are placed in "value" should be verified if they are less than SUPLA_CHANNELVALUE_SIZE
ASSERT_LE(sizeof(TDSC_RollerShutterValue), (unsigned int)SUPLA_CHANNELVALUE_SIZE);

Structures which are placed in "Config" should be verified if they are less than SUPLA_CHANNEL_CONFIG_MAXSIZE
  ASSERT_LE(sizeof(TSD_ChannelConfig_FacadeBlind),
            (unsigned int)SUPLA_CHANNEL_CONFIG_MAXSIZE);